README.md

```diff
@@ -0,0 +1,39 @@
+# Defiant ZMK
+
+This Repository contains the ZMK configuration and keymap for the Defiant Keyboard
+
+## Quickstart
+
+Fork this repository. You probably want to change settings/keymaps for your version of the keyboard.
+
+After that, enable actions on this repository.\
+After Pipeline completion, the firmware will be automatically built.
+<img src="./assets/img/firmwareDownload.png">
+
+Download the firmware. Connect one side of the keyboard with your computer.
+
+Note: If the storage device does not show up, please try to reset the keyboard.
+
+The keyboard will show up as a storage device.
```

Comment:
you might want to add that the board only shows up as storage after pressing reset twice